### DISCUSSION TITLE
Student Application Rejection

### DISCUSSION THREAD

**Discussion by yasiruviyara** [community/community#145091] [GitHub Education]
### Hi everyone,

Hi everyone,

I recently applied for the GitHub Student Developer Pack, but my application was not approved. I am a current university student and believe I meet all the eligibility requirements. However, I may have missed something or submitted incorrect documentation. I would appreciate any guidance from the community to help me resolve this.  I have tried multiple times, submitting my university ID (front and back), enrollment letter, and certificate as proof of my student status.

Best regards,
Yasiru Viyara

![image](https://github.com/user-attachments/assets/b6544407-ebc7-4499-bd19-e8dda979c345)

**Comment by FeiSuoerDe** [comment]
It took me 19 attempts to succeed....

**Reply by HashirSaudKhan** [reply]
What was the major mistake you were making that caused it to take 19 attempts?

**Reply by FeiSuoerDe** [reply]
I don't know, maybe I should compare my personal information with real documents? I'm not sure if this leads to privacy issues. But I used the same file to shoot and constantly refine my personal information during the process, and I'm not sure if that has anything to do with my success. It is clear that the official is not good enough in this regard, such as the front camera, the annoying waiting...

**Reply by yasiruviyara** [reply]
My application was approved after 9 attempts

**Reply by FeiSuoerDe** [reply]
congratulations！
